HGustavs/LenaSYS
# Refactored two CSS class names

## How to test:

When testing, if possible, try both dark and light theme since the refactoring affects both css-files. Additionally, you might want to test it with different screen sizes to make sure the css has been correctly applied after refactoring. For each popup window, make sure functionalities such as tooltips, validation, date-selection and buttons still work as intended. Although, it is important to think about that some of the functionality is kind of broken to begin with so if possible, compare the pull request solution to the weekly branch. Below, the different pages are listed with their respective popup windows.  

courseed.php
- Edit server settings
- Create new course
- Edit course

sectioned.php
- Edit course version (cogwheel in navbar)
- New course version (plus sign in navbar)
- Github repo(download button to the right of lock)

duggaed.php
- Edit dugga (the window is actually for creating a dugga, press big “+” button

fileed.php
- Try all of the options in the picture below
<img width="279" alt="Pasted Graphic 5" src="https://github.com/HGustavs/LenaSYS/assets/129295853/84550d78-1a18-40e6-944f-74d54524d168">

accessed.php
- Try all of the options in the picture below:

<img width="266" alt="Pasted Graphic 6" src="https://github.com/HGustavs/LenaSYS/assets/129295853/d38916ee-3e62-4523-9e58-7bb383fa98f3">

Further testing surrounds manually searching for references of “loginBox” and “loginBoxheader” to see if I have missed any. Note that I have ONLY changed the id and class name of “loginBox” and “loginBoxheader”, and not “loginBoxtr” and similar values that end in different letters. To do this, use Ctrl + Shift + F, or Cmd + Shift + F on Mac.
